ext/CEGUI/IrrlichtBaWTexture.cpp

```diff
@@ -5,80 +5,35 @@ using namespace irr;
 using namespace ext;
 using namespace cegui;
 
-IrrlichtBaWTexture::IrrlichtBaWTexture(irr::core::smart_refctd_ptr<irr::IrrlichtDevice> _device)
-    : device(_device), bufferRowLength(0)
-{
-
-}
+static constexpr std::string_view _IRR_TEXTURE_MUST_EXIST_ = "To perform the function, gpu texture must exist at that point!";
+static constexpr std::string_view _IRR_CEGUI_UNSUPPORTED_FORMAT_ = "Unsupported pixel format detected!";
+static constexpr std::string_view _IRR_WRONG_CASTING_ = "Something went wrong while casting, empty pointer detected!";
+static constexpr std::string_view _IRR_NOT_TIGHTLY_PACKED_ = "Width of memory data passed doesn't pass requirements - it isn't tightly packed!";
 
-IrrlichtBaWTexture::~IrrlichtBaWTexture()
+video::IDriverMemoryBacked::SDriverMemoryRequirements getMemoryRequirements(size_t vulkanReqsSize)
```

Comment:
you don't need this function, because you don't need to create transient buffers (we already have those, they're called the up and downstreaming buffer)